js/src/admin/components/PagesListItem.js

```diff
@@ -14,9 +15,10 @@ export default class PagesListItem extends Component {
     view() {
         const page = this.props.page;
         const url = app.forum.attribute('baseUrl') + '/p/' + page.id() + '-' + page.slug();
+        const badges = page.badges().toArray();
         return (
             <tr key={page.id()}>
-                <th>{page.title()}</th>
+                <th>{page.title()} {badges.length ? m('ul.badges', listItems(page.badges().toArray())) : null}</th>
```

Comment:
I don't think we should be mixing hyperscript & JSX - stick to JSX.

Reply:
How do I do that in JSX ? :sweat_smile:

Reply:
```suggestion
                <th>{page.title()}
                	{badges.length &&
                		<ul className="badges">
                			{listItems(page.badges().toArray()}
                		</ul>
                </th>
```

Reply:
Fixed! thanks :hugs: